rdr_service/gcloud_functions/retention_eligible_import_function/main.py

```diff
@@ -42,10 +42,14 @@ def created(self):
             "upload_date": self.event.timeCreated,
         }
 
-        _logger.info("Pushing cloud task...")
```

Comment:
Why are these cloud function changes in this PR? Normal process is to branch off of devel and apply your changes to a new branch.